### ISSUE TITLE
Search editor revealDefinition resolves to an invalid path

### ISSUE THREAD

**Issue by danyim** [microsoft/vscode#102742]
- VSCode Version: 1.47.2
- OS Version: macOS Mojave 10.14.6

Steps to Reproduce:

1. Open a search editor window with a query via `search.action.openNewEditor`
2. Use reveal definition on an entry `editor.action.revealDefinition`
3. You should see an error like the following:
```
Unable to open 'types.ts': Unable to read file '/Users/user/project/folder/folder/src/types.ts' (Error: Unable to resolve non-existing file '/Users/user/project/folder/folder/src/types.ts').
```

Although the path for `/Users/user/project/folder/src/types.ts` (without an extra `/folder`) exists, the issue is that VSCode inserts a duplicate `folder` upon revealing the definition.

Does this issue occur when all extensions are disabled?: Yes

**Comment by JacksonKearl** [comment]
Thanks for reporting this! 

Do you have a multi-root workspace set up?

**Comment by danyim** [comment]
Yes I do:

```json
	"folders": [
		{
			"name": "Frontend",
			"path": "frontend"
		},
		{
			"name": "Webserver",
			"path": "backend"
		},
		{
			"name": "Mock Backend",
			"path": "tests/mock-backend"
		},
		{
			"name": "E2E",
			"path": "tests"
		}
	],
```

**Comment by JacksonKearl** [comment]
This should be fixed in Insiders, original issue: #100929
